src/juju/types.ts

```diff
@@ -13,8 +13,8 @@ import type ModelManagerV9 from "@canonical/jujulib/dist/api/facades/model-manag
 import type { ModelSLAInfo } from "@canonical/jujulib/dist/api/facades/model-manager/ModelManagerV9";
 import type PingerV1 from "@canonical/jujulib/dist/api/facades/pinger/PingerV1";
 
-import type JIMMV2 from "juju/jimm-facade";
-// See https://github.com/juju/juju/blob/develop/apiserver/params/multiwatcher.go
+import type JIMMV3 from "./jimm/JIMMV3";
```

Comment:
With the above proposed change, I think we could replace this line with `import { type JIMMV3 } from "./jimm";`.